x/poolmanager/router_test.go

```diff
@@ -1030,3 +1118,43 @@ func (suite *KeeperTestSuite) TestSingleSwapExactAmountIn() {
 		})
 	}
 }
+
+// getPoolStrategy returns a function that will setup pools based on the poolType provided
+func (suite *KeeperTestSuite) setupPoolsStrategy(poolType types.PoolType, poolDefaultSwapFee sdk.Dec) (firstEstimatePoolId, secondEstimatePoolId uint64) {
```

Comment:
Please rename other instances as well
```suggestion
func (suite *KeeperTestSuite) setupPools(poolType types.PoolType, poolDefaultSwapFee sdk.Dec) (firstEstimatePoolId, secondEstimatePoolId uint64) {
```